scripts/run_action.mjs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 The Outline Authors
```

Comment:
One question I have is: aren't we reproducing Gulp in some way?
I guess Gulp makes it a lot harder to define tasks in different places, but I don't really know these days.

Reply:
Well, we've been talking about replacing gulp so I figured that was always the plan. If we wanted to lean _into_ gulp we would just use modules to explicitly import all our disparate tasks, though it wouldn't be automatic, especially since `context.require` support is tricky in ESM.

Also gulp doesn't support ESM directly (only through the `esm` package) so I think there might be unintended consequences there.